web/src/lib/components/utilities-page/duplicates/duplicate-asset.svelte

```diff
@@ -38,6 +38,13 @@
         draggable="false"
       />
 
+      <!-- FAVORITE ICON -->
+      {#if !isSharedLink() && asset.isFavorite}
```

Comment:
I don't actually understand why the `!isSharedLink` is here. I copied this snippet from `thumbnail.svelte` so I'm guessing we also need this condition here? Let me know if I should take it out :smile:

Reply:
You don't need it. We hide those types of icons when you are viewing stuff as an anonymous user.

Reply:
okay, I've removed the condition